CHANGES.md

```diff
@@ -10,6 +10,8 @@
 
 <!-- Changes that affect Black's stable style -->
 
+- Fix formatting cells with magic methods and starting or trailing empty lines (#4484)
```

Comment:
```suggestion
- Fix formatting cells in IPython notebooks with magic methods and starting or trailing empty lines (#4484)
```
Otherwise readers who only use Black for formatting regular Python code may be confused.